docs/using-gitbase/functions.md

```diff
@@ -6,6 +6,7 @@ To make some common tasks easier for the user, there are some functions to inter
 
 |     Name     |                                               Description                                                                      |
 |:-------------|:-------------------------------------------------------------------------------------------------------------------------------|
+|`commit_stats(repository_id, [from_commit_hash], to_commit_hash)`|returns the stats between two commits for a repository. If from is empty, it will compare HEAD with the given `to_commit_hash`|
```

Comment:
If `from` is missed then `to` is compared with a parent:
https://github.com/src-d/gitbase/blob/494e149875f5ea4bfe012051aeb59b662d1e64cd/internal/commitstats/commit_stats.go#L40

Reply:
ooopsie